modules/ROOT/pages/servicenow/servicenow-reference.adoc

```diff
@@ -3,107 +3,124 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-== Configuration
+== Configurations
+---
+[[config]]
+=== Config
 
-Default configuration.
++++
+Default configuration
++++
 
+==== Parameters
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
 |Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
-| Connection a| Basic Connection
- | The connection types that can be provided to this configuration. | | x
-| Expiration Policy a| ExpirationPolicy |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform  expires the instance at the exact moment that it becomes eligible. The runtime  actually purges the instances when it sees it fit. |  |
+| Connection a| <<config_basic-connection, Basic Connection>>
+ | The connection types to provide to this configuration. | | x
+| Expiration Policy a| <<ExpirationPolicy>> |  +++Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime purges the instances when it sees fit.+++ |  | 
```

Comment:
Is there anything more specific to say instead of "when it sees fit"? Maybe something like "The runtime purges the instances based on internal criteria"?